config/locales/en.yml

```diff
@@ -154,6 +155,7 @@ en:
       links:
         badge: Badge
         bugs: Bug Tracker
+        changelog: Change Log
```

Comment:
I'd suggest sticking to one-word `Changelog`. We've had many discussions over on http://keepachangelog.com/ about this and it seems like the most correct in-domain (software) spelling.